packages/knowledge/src/i18n/zh-CN.ts

```diff
@@ -279,9 +279,9 @@ export const zhCN: TranslateList = {
       def: 'TBD',
     },
     treemap: {
-      name: '',
+      name: '树图',
       alias: [],
```

Comment:
```suggestion
      alias: ['矩形树图'],
```

ref：http://tuzhidian.com/chart?id=5c56e2434a8c5e048189c6a5